src/lib/containers/table/SynapseTable.tsx

```diff
@@ -523,10 +529,11 @@ export default class SynapseTable extends React.Component<
     const isShowingAccessColumn: boolean | undefined =
       showAccessColumn && this.state.isEntityView
     const isLoggedIn = !!this.context.accessToken
+    const containsGroupBy = isGroupBy(lastQueryRequest.query.sql)
```

Comment:
Do not show the special download columns if the sql contains a "group by", as this changes what a row represents.